src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.html

```diff
@@ -0,0 +1,34 @@
+<p-dialog visible="true" [draggable]="false" [modal]="true" [dismissableMask]="true" (onHide)="hideDialog()">
+    <p-header>
+        {{ dotMessageService.get('editpage.layout.theme.header') }}
+    </p-header>
+
+    <dot-site-selector #siteSelector class="" (change)="siteChange($event)" [archive]="false"></dot-site-selector>
+    <div class="dot-theme-search-box">
+        <i class="fa fa-search searchable-dropdown__search-icon"></i>
+        <input #searchInput pInputText placeholder="{{ dotMessageService.get('editpage.layout.theme.search') }}" type="text" class="searchable-dropdown__search-field"/>
+    </div>
+    <div class="dot-themes-container">
+        <p-dataGrid #dataGrid
+            [value]="themes"
+            [paginator]="true"
+            [rows]="paginatorService?.paginationPerPage"
+            [totalRecords]="paginatorService?.totalRecords"
+            [lazy]="true"
+            (onLazyLoad)="paginate($event)">
+            <ng-template let-theme pTemplate="item" >
+                <div class="dot-theme-item ui-g-3" (click)="selectTheme(theme)" [ngClass]="{'active': theme.inode === current?.inode}">
+                    <svg width="150" height="150" viewBox="0 0 150 150" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M150 133.333V16.6667C150 7.5 142.5 0 133.333 0H16.6667C7.5 0 0 7.5 0 16.6667V133.333C0 142.5 7.5 150 16.6667 150H133.333C142.5 150 150 142.5 150 133.333ZM45.8333 87.5L66.6667 112.583L95.8333 75L133.333 125H16.6667L45.8333 87.5Z"/>
+                    </svg>
+                    <span class="dot-theme-item__name">{{ theme.name }}</span>
+                </div>
+            </ng-template>
+        </p-dataGrid>
+    </div>
```

Comment:
maybe it should be a component itself

Reply:
We already have the `<dot-dialog>` but is not merged, I think we can use it here.

Reply:
I meant the 'dot-themes-container'

Reply:
If we are going to use the theme selector in a different presentation than a pop-up, in other parts of the app, it make sense. @fmontes your call.